src/Data/Fuzzy/Fuzzy.purs

```diff
@@ -0,0 +1,222 @@
+module Data.Fuzzy where
```

Comment:
This applies to the module in general -- I could really use some comments to explain what things like `Fuzzy a` and `Result` are. It's especially true for things that are string-based like `Result`. Nothing too fancy, but a little type and function documentation inline will help me when say using this in the typeahead to ensure I know how to use it properly.

Reply:
I'm finding it tough to understand where the 'forgiving' filtering / ranking is happening and how to modify that. Is there a specific place in the code where this is happening and say a knob can be turned to adjust it?